moja-kuhinjica-frontend/src/components/menu/MenuItem.tsx

```diff
@@ -47,7 +47,7 @@ export const MenuItem = ({
                 }
             >
                 <Image
-                    src={image ? image : pic}
+                    src={image ? image : mealDefault}
```

Comment:
try like this src={image ?? mealDefault}

Reply:
double question marks won't work for empty strings, like in your case, so all good